conans/client/tools/oss.py

```diff
@@ -202,69 +202,69 @@ class _OSVERSIONINFOEXW(ctypes.Structure):
     def get_debian_version_name(version):
         if not version:
             return None
-        elif version.major() == "8.Y.Z":
+        elif version.major == 8:
             return "jessie"
-        elif version.major() == "7.Y.Z":
+        elif version.major == 7:
             return "wheezy"
-        elif version.major() == "6.Y.Z":
+        elif version.major == 6:
             return "squeeze"
-        elif version.major() == "5.Y.Z":
+        elif version.major == 5:
             return "lenny"
-        elif version.major() == "4.Y.Z":
+        elif version.major == 4:
             return "etch"
-        elif version.minor() == "3.1.Z":
+        elif version.minor == 1:
```

Comment:
Don't you want to compare major version as well?
```suggestion
        elif (version.major, version.minor) == (3, 1):
```